javascript/javascript1/week4/js-week4.js

```diff
@@ -0,0 +1,111 @@
+let userName="";
+let toDoList=[];
```

Comment:
Hi!😀 Always prefer to use **const** variables instead of **let** ones. Here's a [nice article](https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa) that explains this further.